src/core/primitives/Dialog/fragments/DialogPrimitivePortal.tsx

```
@@ -0,0 +1,23 @@
+'use client';
+import React from 'react';
+import Floater from '~/core/primitives/Floater';
+
+export type DialogPrimitivePortalProps = {
+  children: React.ReactNode;
+};
+
+const DialogPrimitivePortal = ({ children }: DialogPrimitivePortalProps) => {
+    const fallback = document && document.body;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)